worker.js

```diff
@@ -23,7 +23,13 @@ const PAGE_DESCRIPTION =
 /* Step 4: enter a Google Font name, you can choose from https://fonts.google.com */
 const GOOGLE_FONT = "Rubik";
 
-/* Step 5: enter any custom scripts you'd like */
+/* Step 5: Add a custom favicon (your Notion avatar URL works too) */
+const CUSTOM_AVATAR = "";
```

Comment:
Hello, this section manually works for me,


pd: the only thing that does not come out to the main script?

Reply:
Sorry, could you tell me how it worked? I rewrote it but cloudflare said script can't be deployed. Is there anything wrong with these code?

/* Step 5: Add a custom favicon (your Notion avatar URL works too) */
const CUSTOM_AVATAR = "<link href="https://uploads-ssl.webflow.com/5f66a66ce4b959957c4e8057/60ec40cd1484673a14eeb4b9_chiali_favicon.png" rel="shortcut icon" type="image/x-icon">";

Reply:
/* Step 5: Add a custom favicon (your Notion avatar URL works too) */
const CUSTOM_AVATAR = "<link href="https://uploads-ssl.webflow.com/5f66a66ce4b959957c4e8057/60ec40cd1484673a14eeb4b9_chiali_favicon.png" rel="shortcut icon" type="image/x-icon">";

Reply:
```suggestion
const CUSTOM_AVATAR = "<link href="https://uploads-ssl.webflow.com/5f66a66ce4b959957c4e8057/60ec40cd1484673a14eeb4b9_chiali_favicon.png" rel="shortcut icon" type="image/x-icon">";
```